redbot/cogs/mod/names.py

```diff
@@ -91,7 +91,7 @@ class ModInfo(MixinMeta):
         special_date = datetime(2016, 1, 10, 6, 8, 4, 443000)
         is_special = user.id == 96130341705637888 and guild.id == 133049272517001216
 
-        roles = sorted(user.roles)[1:]
+        roles = user.roles[-1:1:-1]
```

Comment:
I'm so sorry, my prior suggestion had a logical failure in it.
```suggestion
        roles = user.roles[-1:0:-1]
```